README.md

```diff
@@ -2,6 +2,39 @@
 
 [![Curseforge](https://cf.way2muchnoise.eu/full_565238_downloads.svg)](https://www.curseforge.com/minecraft/modpacks/gregtech-expert-2) [![Curseforge](https://cf.way2muchnoise.eu/versions/For%20MC_565238_latest.svg)](https://www.curseforge.com/minecraft/modpacks/gregtech-expert-2) <a title="Join us on Discord!" href="https://discord.gg/xBwHpZyZdW"><img src="https://img.shields.io/discord/945647524855812176?label=GTExpert%20Discord&amp;logo=Discord&amp;style=?flat" alt="Discord"/></a> [![License: LGPL v3](https://img.shields.io/badge/License-LGPL_v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
 
+## Modpack Version Information
+### All Versions
+#### en_us
+* Major version upgrades in the following categories are not compatible.
+
+#### ja_jp
+* 下記区分の大きなバージョンアップは互換性はありません。
+
+### v1.x.x
+#### en_us
+* Stable version as of 10/28/2022 released to CurseForge. Currently, v2 is under development, and bug fixes in v1 will be included in v2.
```

Comment:
Its the most stable version available on curseforge, using old version of GTCEu. Since the dev team are focusing on v2 development, this version will unlikely receive supports.